package.json

```diff
@@ -85,10 +88,14 @@
         }
     },
     "dependencies": {
+        "bootstrap-italia": "1.3.3",
         "prop-types": "^15.7.2",
         "react-select": "^3.0.4",
         "react-transition-group": "^2.3.1",
         "reactstrap": "^8.0.0",
+        "typeface-lora": "^0.0.72",
+        "typeface-roboto-mono": "^0.0.75",
+        "typeface-titillium-web": "^0.0.72",
```

Comment:
I added these, so that in any third-party projects consuming `design-react-kit` these libraries are already available just by importing them (as they will be available in the `node_modules` folder).

Reply:
Smart move! Will prevent the API calls to the search giant.